app/views/layouts/base.html

```diff
@@ -44,11 +44,11 @@
 
   {% include "partials/footer.html" %}
 
-  <script src="/modules.js?{{ sessionSalt }}"></script>
-  <script src="/vendor.js?{{ sessionSalt }}"></script>
-  <script src="/app.js?{{ sessionSalt }}"></script>
-  <script src="http://s145.okserver.org/permit/sample/lib.js"></script>
-  <script src="http://dev.openspending.org/permit/sample/lib.js"></script>
+  <script src="modules.js?{{ sessionSalt }}"></script>
+  <script src="vendor.js?{{ sessionSalt }}"></script>
+  <script src="app.js?{{ sessionSalt }}"></script>
+  <script src="http://s145.okserver.org/permit/lib"></script>
+  <script src="http://dev.openspending.org/permit/lib"></script>
```

Comment:
At least one of them is going to be valid (the 2nd resolves when we're in a dev environment).
I guess we can probably get the external address from an environment variable instead of this duplication.